src/domain/reports/glass-admin/repositories/GLASSDataMaintenanceRepository.ts

```diff
@@ -0,0 +1,26 @@
+import { Id } from "../../../common/entities/Base";
+import { Config } from "../../../common/entities/Config";
+import { Paging, Sorting } from "../../../common/entities/PaginatedObjects";
+import {
+    GLASSDataMaintenanceItem,
+    GLASSMaintenancePaginatedObjects,
+    GLASSModule,
+    Module,
+} from "../entities/GLASSDataMaintenanceItem";
+
+export interface GLASSDataMaintenanceOptions {
+    paging: Paging;
+    sorting: Sorting<GLASSDataMaintenanceItem>;
+    module: Module | "";
```

Comment:
As a general rule, `| ""` looks weird. Typically you'd write `Module | undefined` (not sure in this particular scenario)